Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -4456,16 +4398,12 @@ - (BOOL)_allowsViewportShrinkToFit
 
 - (NSData *)_dataForDisplayedPDF
 {
-    if (![self _isDisplayingPDF])
-        return nil;
-    return [_customContentView web_dataRepresentation];
+    return nil;
 }
 
 - (NSString *)_suggestedFilenameForDisplayedPDF
```

Comment:
Should we file some bugs to remove this SPI?

Reply:
Yes, good idea.

Reply:
Remove -_dataForDisplayedPDF and -_suggestedFilenameForDisplayedPDF SPIs
https://bugs.webkit.org/show_bug.cgi?id=296719